docs/how-to-use-playground.stories.mdx

```diff
@@ -0,0 +1,26 @@
+import { Meta, Description } from '@storybook/addon-docs';
+import playgroundLandingPage from './images/playground-landing-page.png'
+
+<Meta
+  title="Documentation/How To Use Playground"
+  parameters={{
+    viewMode: 'docs',
+    previewTabs: {
+      canvas: {
+        hidden: true,
+      },
+    },
+  }}
+/>
+
+# How to use playground?
+Playground is a demonstration area for developers. It stays under the `playground` folder in the root. It helps you to test your features & components by eliminating the need for stories.
+
+<bl-alert variant="warning" caption="Note">You should not commit the changes in the playground folder.</bl-alert>
+
+You can run it with `npm run serve`, then it builds project and run the build in `watch mode` at port `8000`. You can either create a `foo.html` file and work on it or simply use `template.html`.
+It looks like this,
+
+<img src={playgroundLandingPage}></img>
+
+Every change in your `html` file will be served immediately. However, if you change your component, you should rebuild the project by using `npm run build`.
```

Comment:
> However, if you change your component, you should rebuild the project by using `npm run build`.

That's not needed in this mode. `npm run serve` already works on watch mode for whole build process.

Reply:
🤔